src/js/jsx/shared/TextInput.jsx

```diff
@@ -406,68 +406,43 @@ define(function (require, exports, module) {
                 this._suppressMouseUp = false;
             }
         },
-        
-        /**
-         * When TextInput is disabled, prevent it from scrolling horizontally 
-         * by preventing the default wheel action if there is a non-zero deltaX 
-         * and instead firing a new wheel action with deltaX set to 0.
-         *
-         * @private
-         * @param {SyntheticEvent} event
-         */
-        _handleWheel: function (event) {
-            if (this.props.preventHorizontalScrolling && event.deltaX) {
-                var nativeEvent = event.nativeEvent,
-                    domEvent = new window.WheelEvent(event.type, {
-                        deltaX: 0.0,
-                        deltaY: nativeEvent.deltaY
-                    });
-
-                event.preventDefault();
-                event.target.dispatchEvent(domEvent);
-            }
-        },
 
         render: function () {
             var className = classnames(this.props.className, this.props.size);
 
             if (this.state.editing || !this.props.doubleClickToEdit) {
                 return (
-                    <input
-                        type="text"
-                        ref="input"
-                        spellCheck="false"
-                        value={this.state.value}
+                    <div
```

Comment:
Is the div around the input really necessary? It definitely seems like layout time is sensitive to extra complexity like this, especially when multiplied by ~1000 `TextInput` instances.